docs/examples/policies/sbom/sbom-syft.yaml

```diff
@@ -16,14 +16,15 @@ apiVersion: workflowcontract.chainloop.dev/v1
 kind: Policy
 metadata:
   name: made-with-syft
+  description: Verifies that the SPDX was created with Syft
+  annotations:
+    category: sbom
 spec:
   type: SBOM_SPDX_JSON
```

Comment:
this is specifically about spdx, should we call it `spdx-sbom-made-with-syft` and maybe add one for cycloinedx json?